evm/src/contracts/CircleIntegration/Storage.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.22;
+
+/**
+ * struct State {
+ *         // state.slot := 0x0
+ *         uint16 chainId;
+ *         uint8 wormholeFinality;
+ *         uint32 localDomain;
+ *         address wormhole;
+ *         uint16 governanceChainId;
+ *
+ *         // state.slot := 0x1
+ *         bytes32 governanceContract;
+ *
+ *         // state.slot := 0x2
+ *         address circleBridgeAddress;
+ *
+ *         // state.slot := 0x3
+ *         address circleTransmitterAddress;
+ *
+ *         // state.slot := 0x4
+ *         address circleTokenMinterAddress;
```

Comment:
None of these are actually consuming storage slots. Immutables are inlined into bytecode at deployment time and constants are inlined also